functorch/csrc/init.cpp

```diff
@@ -398,7 +388,6 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
   m.def("maybe_get_level", &at::functorch::maybe_get_level);
   m.def("maybe_get_bdim", &at::functorch::maybe_get_bdim);
   m.def("current_level", &at::functorch::currentLevel);
-  m.def("unwrap_batchedtensor", &at::functorch::unwrapTensorAtCurrentLevel);
```

Comment:
Needed to delete because this was causing missing symbols. Also, this function isn't actually used